tests/sources/xml/test_dspace_dim.py

```diff
@@ -472,3 +472,308 @@ def test_get_file_formats_transforms_correctly_if_fields_missing():
+        qualifier="spatial">Central equatorial Pacific Ocean</dim:field>
+        """
+    )
+    assert DspaceDim.get_locations(source_record) == [
+        timdex.Location(value="Central equatorial Pacific Ocean")
+    ]
+
+
+def test_get_locations_transforms_correctly_if_fields_blank():
+    source_record = create_dspace_dim_source_record_stub(
+        '<dim:field mdschema="dc" element="coverage" qualifier="spatial" />'
+    )
+    assert DspaceDim.get_locations(source_record) is None
+
+
+def test_get_locations_transforms_correctly_if_fields_missing():
+    source_record = create_dspace_dim_source_record_stub()
+    assert DspaceDim.get_locations(source_record) is None
+
+
+def test_get_notes_success():
+    source_record = create_dspace_dim_source_record_stub(
+        """
+        <dim:field mdschema="dc"
+        element="description">Author Posting. © The Author(s), 2008.</dim:field>
+        <dim:field mdschema="dc" element="description"
+        qualifier="embargo">2026-01</dim:field>
+        """
+    )
+    assert DspaceDim.get_notes(source_record) == [
+        timdex.Note(value=["Author Posting. © The Author(s), 2008."]),
+        timdex.Note(value=["2026-01"], kind="embargo"),
+    ]
+
+
+def test_get_notes_transforms_correctly_if_fields_blank():
+    source_record = create_dspace_dim_source_record_stub(
+        '<dim:field mdschema="dc" element="description" />'
+    )
+    assert DspaceDim.get_notes(source_record) is None
+
+
+def test_get_notes_transforms_correctly_if_fields_missing():
+    source_record = create_dspace_dim_source_record_stub()
+    assert DspaceDim.get_notes(source_record) is None
+
+
+def test_get_publishers_success():
+    source_record = create_dspace_dim_source_record_stub(
+        """
+        <dim:field mdschema="dc"
+        element="publisher">Woods Hole Oceanographic Institution</dim:field>
+        """
+    )
+    assert DspaceDim.get_publishers(source_record) == [
+        timdex.Publisher(name="Woods Hole Oceanographic Institution")
+    ]
+
+
+def test_get_publishers_transforms_correctly_if_fields_blank():
+    source_record = create_dspace_dim_source_record_stub(
+        '<dim:field mdschema="dc" element="publisher" />'
+    )
+    assert DspaceDim.get_publishers(source_record) is None
+
+
+def test_get_publishers_transforms_correctly_if_fields_missing():
+    source_record = create_dspace_dim_source_record_stub()
+    assert DspaceDim.get_publishers(source_record) is None
+
+
+def test_get_related_items_success():
+    source_record = create_dspace_dim_source_record_stub(
+        """
+        <dim:field mdschema="dc" element="relation"
+        >A low resolution version of this movie was published.</dim:field>
+        <dim:field mdschema="dc" element="relation" qualifier="ispartofseries"
+        >International Association of Aquatic and Marine Science</dim:field>
+        <dim:field mdschema="dc" element="relation" qualifier="uri"
+        >https://doi.org/10.1002/2016JB013228</dim:field>
```

Comment:
This is a very minor nit, not a required or even suggested change, but I think personally I would find these kind of stubs easier to read if the elements were one line?  

e.g.,
```python
source_record = create_dspace_dim_source_record_stub(
    """
    <dim:field mdschema="dc" element="relation">A low resolution version of this movie was published.</dim:field>
    <dim:field mdschema="dc" element="relation" qualifier="ispartofseries">International Association of Aquatic and Marine Science</dim:field>
    <dim:field mdschema="dc" element="relation" qualifier="uri">https://doi.org/10.1002/2016JB013228</dim:field>
    """
)
```

I understand this breaks line length linting... but the readability might be worth it sometimes.

Again, not requesting a change -- particularly as it's probably fairly common across transforms -- but noting to propose that it's an option when we want.

Reply:
We can talk about that, not a big fan of sidescrolling, but not sure if I'm the only one so a later discussion is warranted!

Reply:
I think I prefer side-scrolling 🤔. If the XML tags themselves are long, I would also prefer:

```
<tag> 
   [some long string]
</tag>
```

Reply:
Got it, yes, let's discuss further!